pom.xml

```diff
@@ -881,8 +881,10 @@
             331,
             332,
             333,
+            334,
             340,
             341,
+            342,
             350
         </noSnapshotScala213.buildvers>
         <snapshotScala213.buildvers>
```

Comment:
Need to add 351 to `snapshotScala213.buildvers` as well

Reply:
This is for 24.02 release build, snapshot shim doesn't matter.

Reply:
> Need to add 351 to `snapshotScala213.buildvers` as well

Good catch1

Reply:
> > Need to add 351 to `snapshotScala213.buildvers` as well
> 
> We do not support 3.5.1 in 24.02, right?

Yes, we only build/test against spark-3.5.1-SNAPSHOT